src/lib/repositories/environment-tab.repository.ts

```diff
@@ -33,7 +33,7 @@ export class EnvironmentTabRepository {
     if (existedTab) {
       await existedTab.incrementalUpdate({ $set: { isActive: true } });
     } else {
-      await RxDB.getInstance().rxdb.environmentTab.insert(environmentTab);
+      await RxDB.getInstance().rxdb.environmenttab.insert(environmentTab);
     }
```

Comment:
we can follow camelCase convention

Reply:
RxDB Collection names must match the following regex: ^[a-z][a-z0-9]*$
Documenation -https://rxdb.info/rx-collection.html